examples/gridViewExample/src/main/java/io/realm/examples/realmgridview/GridViewExampleActivity.java

```diff
@@ -95,23 +103,34 @@ protected void onDestroy() {
             return null;
         }
 
-        JsonParser parser = new JsonParser();
-        JsonArray jsonArray = parser.parse(new InputStreamReader(stream)).getAsJsonArray();
+        // GSON can parse the data.
+        // Note there is a bug in GSON 2.3.1 that can cause it to StackOverflow when working with RealmObjects.
```

Comment:
Not really as the solution for the bug is just below, but perhaps it should reworded to be more clear about that.